<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palm Coin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="platformLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Platform:</string>
    </property>
   </widget>
   <widget class="QLabel" name="platformInfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLab">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>User name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameInfo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frozenLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Frozen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceInfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(143, 218, 22);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="frozenInfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>531</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="marketTab">
     <attribute name="title">
      <string>Market</string>
     </attribute>
     <widget class="QLabel" name="asksLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Asks</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="asksDepth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>521</width>
        <height>211</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Volume</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="bidsLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bids</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="bidsDepth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>521</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Volume</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="freshButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fresh</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="manualRadio">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="autoRadio">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="periodSpin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="periodLab">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QComboBox" name="currencyPaireCombo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>69</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>BTC/CNY</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="deep_spinner">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Deep:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="orderTab">
     <attribute name="title">
      <string>Orders</string>
     </attribute>
     <widget class="QPushButton" name="freshButtono">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="buyList">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>521</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date-Time</string>
       </property>
      </column>
      <item row="0" column="3">
       <property name="text">
        <string>gfsgfs</string>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="periodSpino">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="sellLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sell</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="sellList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>521</width>
        <height>211</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date-Time</string>
       </property>
      </column>
     </widget>
     <widget class="QRadioButton" name="autoRadioo">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="manualRadioo">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="periodLab2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="buyLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Buy</string>
      </property>
     </widget>
     <widget class="QComboBox" name="currencyPaireComboo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>69</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>BTC/CNY</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
